src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabControl.cs

```diff
@@ -870,7 +870,7 @@ public event TabControlCancelEventHandler? Selecting
     }
 
     /// <summary>
-    ///  Occurs after a tabpage is selected as the top tabPage.
+    ///  Occurs after a tabPage is selected as the top tabPage.
```

Comment:
```suggestion
    ///  Occurs after a TabPage is selected as the top TabPage.
```

THere are more occurrences of camel-casing - tabPage in this, please spell them as TabPage.